src/box/lua/config/cluster_config.lua

```diff
@@ -316,10 +341,17 @@ return schema.new('cluster_config', record_from_fields({
             -- are going to be merged into the main config
             -- (i.e. the validation is performed after the version
             -- comparison).
-            key = schema.scalar({type = 'string'}),
-            value = schema.scalar({type = 'any'}),
+            key = schema.scalar({
+                type = 'string',
+                description = descriptions['conditional.*']
+            }),
+            value = schema.scalar({
+                type = 'any',
+                description = descriptions['conditional.*.*']
+            }),
```

Comment:
Why two nesting levels? Looks like the path should be just `conditional.*`.

Reply:
Because it's a map inside an array, and we got a double `*`.